golang/static/js/peering.js

```diff
@@ -73,9 +73,16 @@ class Peering {
     let video = document.createElement("video")
     video.id = peerId
     video.autoplay = true
+    video.controls = true
+    video.muted = true
+    video.poster = 'https://upload.wikimedia.org/wikipedia/commons/d/d3/WUERFEL5_0-_bis_5-dimensionale_Wuerfelanaloge.png'
 
     this.videosElm.appendChild(video)
 
+    setTimeout(() => {
+      video.play()
+    }, 1000)
+
```

Comment:
Any more details on the race condition that requires this? Are we not emitting an event somewhere? Maybe the issue is the lack of handling of the `negotiationneeded`, which the frontend should be handling correctly now.